examples/pinn_pde_solver/pinn_diffusion_pde_solver.cpp

```diff
@@ -39,7 +40,7 @@ const int ADAM_STEPS = 1000;
 const int MAX_STEPS = 5000;
 // criteria for stop training.
 // When loss is at 1.x~e-5, it will stop degrading even the training continues
-const float TARGET_LOSS = 2e-5;
```

Comment:
Is this change for a performance improvement? Or does it simply not terminate if TARGET_LOSS is set too low?

Reply:
It's the latter one. Even set to 2e-5, sometimes it would not stop.